<launch>

<!-- Default folder with maps-->

<arg name="folder" default="$(env HOME)/stroll"/>

<!-- Distance Measurement Node -->
  <node pkg="stroll_bearnav" type="odometry_monitor_node" name="odometry_monitor"/>

<!-- feature extraction -->
  <node pkg="stroll_bearnav" type="feature_extraction_node" name="feature_extraction">
	  <remap from="image" to="/stereo/left/image_raw"/>
  </node>

<!-- mapping node -->
<node pkg="stroll_bearnav" type="mapper_node" name="mapper" output="screen">
  <remap from="image" to="/stereo/left/image_raw"/>
  <param name="folder" value="$(arg folder)" />
  <param name="angularSpeed" value="0.2" />
</node>

</launch>
